foreman/client.py

```diff
@@ -526,7 +526,7 @@ class Foreman(object):
     def __init__(self, url, auth=None, version=None, api_version=None,
                  use_cache=True, strict_cache=True, timeout=60,
                  timeout_post=600, timeout_delete=600, timeout_put=None,
-                 verify=False):
+                 verify=False, loglevel="DEBUG"):
```

Comment:
mm, I don't think this is a good idea, as you are changing the loglevel of the logger (that is a singleton) you are changing it for any other instance and method that uses the same logger too.

This should be a module level method instead and not force it on each init, just leave the decision to the user if he wants to set any loglevel to that specific logger (or just let it inherit the config from the root logger)